File: Programmers/Lv.1/가운데글자가져오기.py
def solution(s):
    answer = ''
    center = len(s) // 2
    
    if len(s) % 2 != 0:
        answer = s[center]
    else:
        answer = s[center - 1: center + 1]
        
    return answer

# def solution(str):
#     return str[(len(str) - 1) // 2 : len(str) // 2 + 1]

File: Programmers/Lv.1.Kotlin/가운데글자가져오기.kt
class Solution {
    fun solution(s: String): String {
        var answer = ""
        
        if (s.length % 2 == 0) {
            answer = s.get((s.length / 2) - 1).toString() + s.get(s.length / 2).toString()
        } else {
            answer = s.get(s.length / 2).toString()
        }
        
        return answer
    }

    // fun solution(s: String): String =
    //     s.slice((s.length - 1) / 2 .. s.length / 2)
}